### PR TITLE
Ext analytics tweaks

### PR DESCRIPTION
**Pull request by martinraison**
@2is10

### CODE REVIEW

File: packrat/main.js [hunk 1]
```diff
@@ -753,10 +753,24 @@ api.port.on({
       api.tabs.emit(tab, 'hide_external_messaging_intro');
     });
     if (prefs) prefs.showExtMsgIntro = false;
-    mixpanel.track('user_was_notified', {type: source, action: 'click', subaction: action, category: 'extMsgFTUE'});
+    mixpanel.track('user_was_notified', {
+      type: source,
+      action: 'click',
+      subaction: action,
+      channel: 'kifi',
```
Comment (2is10) [inline packrat/main.js:760]: curious what `channel: 'kifi'` signifies. where else do we have `extMsgFTUE`?  ;)

Comment (2is10) [inline packrat/main.js:760]: (wondering if maybe it should be `channel: 'extension'`)

**Comment by 2is10** [comment]
lgtm

File: packrat/main.js (hunk 1 above)
Comment (martinraison) [inline packrat/main.js:760]: I don't think we have `extMsgFTUE` anywhere else, but I think `channel: 'kifi'` matches something else in mixpanel. These conventions are defined by Jen so she probably knows more about them.

**Pull request merged**